File: 59A/59A.py
s = input()
low = 0
up = 0
for i in range(0, len(s)):
    if s[i].islower():
        low += 1
    elif s[i].isupper():
        up += 1
if low < up:
    s = s.upper()
elif low > up:
    s = s.lower()
elif low == up:
    s = s.lower()
print(s)

File: 59A/59A.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    char s[100];
    int lower = 0, upper = 0;
    cin >> s;
    for (int i = 0; i < strlen(s); i++)
    {
        if (isupper(s[i]))
            upper++;
        else if (islower(s[i]))
            lower++;
    }
    if (lower < upper)
    {
        for (int i = 0; i < strlen(s); i++)
            s[i] = toupper(s[i]);
    }
    else if (lower > upper)
    {
        for (int i = 0; i < strlen(s); i++)
        {
            s[i] = tolower(s[i]);
        }
    }
    else if (lower == upper)
    {
        for (int i = 0; i < strlen(s); i++)
        {
            s[i] = tolower(s[i]);
        }
    }
    cout << s << endl;

    return 0;
}